setup.py

```diff
@@ -22,12 +22,12 @@
 
 setup(
     name=NAME,
-    version="1.0.1",
-    author='mbusserolle',
-    author_email='mbussero@gemini.edu',
+    version="2.0.0",
+    author='ncomeau',
+    author_email='ncomeau@uvic.ca',
```

Comment:
Now, we need to decide about where this project should live.  Its true we don't what to bother gemini with mistakes we might be adding, but we also don't want to take credit for code they have written.

Reply:
I'll avoid pushing it to PyPI for now. I know some Dockerfiles will eventually depend on this; perhaps we could bundle a built python wheel (.whl) with our Dockerfiles rather than having them pull the package from PyPI? I'll probably eventually have to talk to Marie at Gemini North someday too...

Reply:
`Dockerfile` can install from a `github` repo if required.

Reply:
Perfect, implemented in https://github.com/ijiraq/gemini_processing/commit/a05258733c2afc26016839a9e290fe0b71b0c1c6 . Now the Dockerfile installs from the "dev" branch of Nifty4Gemini.